code/index.html

```diff
@@ -4,27 +4,200 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <link rel="stylesheet" href="style.css">
+    <style>
+      @import url('https://fonts.googleapis.com/css2?family=Bebas+Neue&display=swap');
+     </style>
   </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
+ 
+<body>
+   
+    
+    <!-- Navbar-->
+   
+    <nav>
+        <div class="nav-content">
+        <div class="nav-logo"><img src="images/whiteicon.png"></div>
+        
+        <div class="nav-links">
+          <a href="#" class="nav-link">About</a>
+          <a href="#" class="nav-link">Coach</a>
+          <a href="#" class="nav-link">Log in</a>
+          <a href="#" class="nav-link">Member</a>
+          <a href="#" class="nav-link">Blog</a>
+        </div>
+    </nav>
```

Comment:
Nice job Maria! :) Really clean and readable code. 
When I resized you site to tablet screen, I saw that logo and nav-links is to close to the border of the browser, maybe if you add some margin to the sides it will look much better?

Reply:
Thank you Darya. I see that I don´t have the responsive breakpoint for tablet in my code. I will fix that :)